src/core/dbus-execute.c

```diff
@@ -1301,6 +1334,58 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "RootImage"))
                 return bus_set_transient_path(u, name, &c->root_image, message, flags, error);
 
+        if (streq(name, "RootImageOptions")) {
+                _cleanup_(mount_options_free_allp) MountOptions *options = NULL;
+                _cleanup_free_ char *format_str = NULL;
+                const char *mount_options;
+                unsigned int partition_number;
```

Comment:
"unsigned" is fine, no need to write out "unsigned int"

Reply:
done